### DISCUSSION TITLE
How to stub useLoaderData for this plugin?

### DISCUSSION THREAD

**Discussion by zxlvera** [JesusTheHun/storybook-addon-remix-react-router#78] [Q&A] [answered]
I got `Cannot destructure property 'env' of 'useLoaderData(...)' as it is undefined.` in my storybook when trying to load a component. Any options to stub/mock `useLoaderData` fn?

**Comment by JesusTheHun** [comment]
To mock data fetchers of any kind, I strongly recommend you to use `msw`. It will operate and the network level and therefore require no change in your codebase.

**Reply by zxlvera** [reply, marked as answer]
Thank you, in the end i followed the solution [here](https://blog.sentiero.digital/how-to-run-remix-link-component-in-storybook)  to create a stub!
